VideoLocker/res/layout/activity_login.xml

```diff
@@ -39,7 +39,7 @@
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
                             android:layout_centerHorizontal="true"
-                            android:contentDescription="@string/app_name"
+                            android:contentDescription="@string/phonetic_app_name"
```

Comment:
can we actually make this `@null`? It's just decorative; we don't need to announce "edX" twice